.github/workflows/full-test-with-codecov.yml

```diff
@@ -48,8 +46,6 @@ jobs:
         with:
           path: ~/spikeinterface_datasets
           key: ${{ runner.os }}-datasets-${{ steps.vars.outputs.HASH_EPHY_DATASET }}
-          restore-keys: |
-            ${{ runner.os }}-datasets
```

Comment:
maybe we should kee this for datasets because it's handled by datalad

Reply:
All right, should be reverted.